templates/server/puppetserver/conf.d/ca.conf.erb

```diff
@@ -7,4 +7,11 @@ certificate-authority: {
 
     # enable the separate CRL for Puppet infrastructure nodes
     enable-infra-crl: <%= @ca_enable_infra_crl %>
+
+    # Setup auto renewal of certs
+    allow-auto-renewal: <%= @server_ca_allow_auto_renewal %>
+    # This value determines the lifetime of the cert if auto-renewal is enabled
+    auto-renewal-cert-ttl: <%= @server_ca_allow_auto_renewal_cert_ttl %>
```

Comment:
These values are not known in the scope, so they don't get replaced. That causes the build to fail:
CI failures are related:
```
  Dec 28 19:30:38 centos9-64-puppet8.example.com puppetserver[2140]: /etc/puppetlabs/puppetserver/conf.d/ca.conf: 14: Expecting close brace } or a comma, got ':' (if you intended ':' to be part of a key or string value, try enclosing the key or value in double quotes)
```

You need to add them to `puppet::server` (and further) as well.